dot_config/nvim/lua/tap/utils/init.lua.tmpl

```
@@ -290,4 +290,28 @@ function utils.termcodes(str)
     return vim.api.nvim_replace_termcodes(str, true, true, true)
 end
 
+--- Attempt to load plugin config, silently fail if there are errors
+---@param name string
+---@return table|nil
+function utils.require_plugin(name)
+    if not string.match(name, '^tap%.plugins%..+') then
+        vim.notify(string.format("Attempted to load non-plugin! [%s]", name),
+                   "error", {title = "require_plugin"})
+        return nil
+    end
+
+    return setmetatable({}, {
+        __index = function(t, k)
+            local ok, plugin = pcall(require, name)
+
+            if not ok then return nil end
```

Wonder if I need to throw an error here rather than return `nil`